### ISSUE TITLE
Feature request: expose the options to disable/enable Tor

### ISSUE THREAD

**Issue by johnpwarren** [brave/browser-laptop#14603]
### Description
I've run into the problem that on some networks I use Tor is banned, but there is no easy configuration to turn off Tor.  There is an error popup that shows up when trying to open a private tab that gives an option to turn off Tor but this really seems like it belongs in the preferences.

Stretch goal on this would be to enable Tor in shields so you can have site by site control.

### Steps to Reproduce
  1. Run Brave on a network that blocks Tor
  2. Wait 5 minutes
  3. Explain to the CyberDefense guys who came to your desk that it's just some new hotshot browser trying to do right

### Brave Version

**about:brave info:**

Name | Version
-- | --
Brave | 0.23.19
V8 | 6.7.288.46
rev | 178c3fb
Muon | 7.1.3
OS Release | 17.6.0
Update Channel | Release
OS Architecture | x64
OS Platform | macOS
Node.js | 7.9.0
Tor | 0.3.3.7 (git-035a35178c92da94)
Brave Sync | v1.4.2
libchromiumcontent | 67.0.3396.87

**Reproducible on current live release:**
Yes

### Additional Information
Tor is a great feature, I'd just like more control over it.

**Comment by srirambv** [comment]
Thinks this will be fixed by https://github.com/brave/browser-laptop/issues/12989. cc: @riastradh-brave

**Comment by johnpwarren** [comment]
@srirambv Thanks!  A temporary fix for me was to kill the tor daemon in the CLI

`killall "tor"`

**Comment by user18x** [comment]
Can i disable tor? Kill a process isn't the best way!

**Comment by davidtemkin** [comment]
@johnpwarren Wouldn't lazy-loading Tor fix this? I.e., rather than add a new preference, we don't start the Tor process until/unless user invokes a Tor private tab.

**Comment by riastradh-brave** [comment]
#12989 won't fix this -- it will just provide another way to reach the Tor network if the default one doesn't work.  Deferring and/or configuring tor startup is under discussion.

**Comment by johnpwarren** [comment]
@davidtemkin I would assume that would be correct.  Still, having detailed control over Tor would be more desirable.  My experience was actually coming into the office after the weekend and learning about Tor traffic from my workstation from the aforementioned CyberDefense guys.  I could replicate it by killing the Tor process, stopping and starting Brave.  I haven't checked yet to see if Brave stops the process on exit yet.

**Comment by johnpwarren** [comment]
Also, it's probably not a good idea for browser updates to happen over Tor (I am now not the only person in the office who has ticked off the CyberDefense guys [ tor traffic laptop-updates.brave.com]).

**Comment by diracdeltas** [comment]
Marking this as non-blocking since https://github.com/brave/browser-laptop/issues/14617 will disable Tor until the user explicitly opens a Tor tab.

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/454
